homework/style.css

```diff
@@ -1,3 +1,178 @@
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  color: red;
+  background: pink;
+  height: 3rem;
+  text-align: center;
+  padding-top: 1rem;
+}
+
+body {
+  width: 768px;
+  margin-left: auto;
+  margin-right: auto;
+  background-color: #d8d8d8c7;
+  font-family: 'Roboto', sans-serif;
+  color: rgba(54, 54, 54, 0.87);
+  margin-top: 0;
+  line-height: 1.5rem;
+}
+
+#header {
+  background: #f96363;
+  color: #ffffffc5;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border-radius: 0.2rem;
+  padding: 8px 16px;
```

Comment:
Hi Ammar - I see that you are using `rem` units which is quite nice but I see that there is some consistency still needed as you are using both `px` and `rem` for margins, paddings, font-sizes and line-heights. Can you please update them? 🙂

Reply:
I changed all of them into rem. :sweat_smile:  Thanks Yash :pray: